cli/bptest/cmd.go

```diff
@@ -71,18 +71,22 @@ var runCmd = &cobra.Command{
 		if err := cobra.ExactArgs(1)(cmd, args); err != nil {
 			return err
 		}
-		if err := isValidTestName(flags.testDir, args[0]); err != nil {
-			return err
-		}
 		return nil
 	},
 	RunE: func(cmd *cobra.Command, args []string) error {
-		intTestDir := flags.testDir
+		intTestDir, err := getIntTestDir(flags.testDir)
+		if err != nil {
+			return err
```

Comment:
We should probably return a human-friendly error here.

Reply:
done